patches/chrome-browser-renderer_context_menu-render_view_context_menu.cc.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/chrome/browser/renderer_context_menu/render_view_context_menu.cc b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+index 78628f4cca9eb1aa6dce06daf0d6cc0b11e212f4..9b34f346d5e112f6f1569dce9fb3070cfa7d2e9a 100644
+--- a/chrome/browser/renderer_context_menu/render_view_context_menu.cc
++++ b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+@@ -1055,6 +1055,10 @@ void RenderViewContextMenu::AppendLinkItems() {
+           IDC_CONTENT_CONTEXT_OPENLINKOFFTHERECORD,
+           is_app ? IDS_CONTENT_CONTEXT_OPENLINKOFFTHERECORD_INAPP
+                  : IDS_CONTENT_CONTEXT_OPENLINKOFFTHERECORD);
++      menu_model_.AddItemWithStringId(
```

Comment:
can you patch header to add `void AppendBraveLinkItems` and call it in `AppendLinkItems` before `AppendOpenWithLinkItems`? Also put the implementation in override .cc
because we might extend `AppendBraveLinkItems` in the future

Reply:
with the subclass comment below, we can put `AppendBraveLinkItems` in subclass and then friend it because we need to access `menu_model_`

Reply:
39a49d92ae72e406f4abc24a678155358b3b7806